public/app/features/plugins/extensions/utils.test.tsx

```diff
@@ -30,17 +29,17 @@ jest.mock('app/features/plugins/pluginSettings', () => ({
   getPluginSettings: () => Promise.resolve({ info: { version: '1.0.0' } }),
 }));
 
-jest.mock('./logs/log', () => {
-  const { createLogMock } = jest.requireActual('./logs/testUtils');
-  const original = jest.requireActual('./logs/log');
+describe('Plugin Extensions / Utils', () => {
+  beforeEach(() => {
+    jest.spyOn(log, 'error').mockImplementation(() => {});
+    jest.spyOn(log, 'warning').mockImplementation(() => {});
+    jest.spyOn(log, 'debug').mockImplementation(() => {});
+  });
 
-  return {
-    ...original,
-    log: createLogMock(),
-  };
-});
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
```

Comment:
The previous solution didn't work as expected as tests where passing for `expect(log.warning).toHaveBeen...` when they should have been  `expect(log.error).toHaveBeen...`